src/Uno.Foundation/Interop/TSInteropMarshaller.wasm.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+using Uno.Foundation;
+
+namespace Uno.Foundation.Interop
+{
+	public static class TSInteropMarshaller
+	{
+		public const UnmanagedType LPUTF8Str = (UnmanagedType)48;
+
+		/// <summary>
+		/// Prints the actual offsets of the structures present in <see cref="WindowManagerInterop"/> for debugging purposes.
+		/// </summary>
+		internal static void GenerateTSMarshallingLayouts()
+		{
+			Console.WriteLine("Generating layouts");
+
+			//foreach (var p in typeof(WindowManagerInterop).GetNestedTypes(System.Reflection.BindingFlags.NonPublic).Where(t => t.IsValueType))
```

Comment:
It's normal that this part is commented? Need it for later or forgot to remove it ?

Reply:
It's useful for debugging :)